src/blockcfg/mock.rs

```diff
@@ -309,6 +321,7 @@ impl std::fmt::Display for Error {
             }
             Error::InvalidSignature(_, _, _) => write!(f, "Input is not signed properly"),
             Error::InvalidTxSignature(_) => write!(f, "Transaction was not signed"),
+            Error::TransactionSumIsNonZero(_, _) => write!(f, "Transaction is not zero"),
```

Comment:
The message leaves out the "sum". Could it be named more concisely: NonZeroBalance?